test/groovy/com/sap/piper/versioning/DockerArtifactVersioningTest.groovy

```diff
@@ -20,7 +20,24 @@ class DockerArtifactVersioningTest extends BasePiperTest{
 
     String passedDir
 
-    JenkinsReadFileRule readFileRule = new JenkinsReadFileRule(this, 'test/resources/versioning/DockerArtifactVersioning')
+    JenkinsReadFileRule readFileRule = new JenkinsReadFileRule(this,
+        ['Dockerfile': ('''|FROM rootImage:1.2.3
+                           |
+                           |USER root
+                           |
+                           |ENV TEST 2.3.4
+                           |''' as CharSequence).stripMargin(),
+         Dockerfile_registryPort : ('''|FROM registry:4444/path/image:1.2.3
+                    |
+                    |USER root
+                    |
+                    ||ENV TEST 2.3.4''' as CharSequence).stripMargin(),
+         Dockerfile_registryPortNoTag :('''|FROM registry:4444/path/image
+                                           |
+                                           |USER root
+                                           |
+                                           |ENV TEST 2.3.4''' as CharSequence).stripMargin()
```

Comment:
I suggest putting such long strings in a method at the end of the class to not clogg up the test setup.

Reply:
For longer files I agree. For shorter files it is IMO a benefit to have the content inline since we can see directly what is going on. Now we have to agree on what is 'short/big' ...